frontend/src/api/api.test.ts

```diff
@@ -1,102 +1,106 @@
-    const pdfFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.PDF }
-    const conversationRequest = createConversationRequestWithUploadedFile(pdfFile)
+  it("formats content correctly when a .pdf is uploaded", async () => {
+    const pdfFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.PDF };
+    const conversationRequest = createConversationRequestWithUploadedFile(pdfFile);
 
-    await conversationApi(conversationRequest, defaultAbortSignal, null)
+    await conversationApi(conversationRequest, defaultAbortSignal, null);
 
-    expect(fetch).toHaveBeenCalledWith('/conversation', {
+    expect(fetch).toHaveBeenCalledWith("/conversation", {
       ...defaultFetchRequest,
-      body: expect.stringContaining('Use the following document in your responses')
-    })
-  })
+      body: expect.stringContaining("Use the following document in your responses"),
+    });
+  });
 
-  it('formats content correctly when a .docx file is uploaded', async () => {
-    const docxFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.DOCX }
-    const conversationRequest = createConversationRequestWithUploadedFile(docxFile)
+  it("formats content correctly when a .docx file is uploaded", async () => {
+    const docxFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.DOCX };
+    const conversationRequest = createConversationRequestWithUploadedFile(docxFile);
 
-    await conversationApi(conversationRequest, defaultAbortSignal, null)
+    await conversationApi(conversationRequest, defaultAbortSignal, null);
 
-    expect(fetch).toHaveBeenCalledWith('/conversation', {
+    expect(fetch).toHaveBeenCalledWith("/conversation", {
       ...defaultFetchRequest,
-      body: expect.stringContaining('Use the following document in your responses')
-    })
-  })
+      body: expect.stringContaining("Use the following document in your responses"),
+    });
+  });
 
-  it('formats content correctly when a .csv file is uploaded', async () => {
-    const csvFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.CSV }
-    const conversationRequest = createConversationRequestWithUploadedFile(csvFile)
+  it("formats content correctly when a .csv file is uploaded", async () => {
+    const csvFile: UploadedFile = { ...defaultUploadedFile, extension: ACCEPTED_FILE_TYPES.CSV };
+    const conversationRequest = createConversationRequestWithUploadedFile(csvFile);
 
-    await conversationApi(conversationRequest, defaultAbortSignal, null)
+    await conversationApi(conversationRequest, defaultAbortSignal, null);
 
-    expect(fetch).toHaveBeenCalledWith('/conversation', {
+    expect(fetch).toHaveBeenCalledWith("/conversation", {
       ...defaultFetchRequest,
-      body: expect.stringContaining('CSV format')
-    })
-  })
-})
+      body: expect.stringContaining("CSV format"),
+    });
+  });
+});
```

Comment:
[sand] outside the scope of this PR, but I wonder if these `formats content correctly when a X file is uploaded` tests can be combined into an `each()`?